### PR TITLE
fix: building mfe image on arm64

### LINKED ISSUE #31: Support for ARM64
Opening a parallel issue to https://github.com/overhangio/tutor/issues/510. I'm currently able to build on an M1 MacBook by adding a few extra build requirements and environment variables:

```diff
diff --git a/tutormfe/templates/mfe/build/mfe/Dockerfile b/tutormfe/templates/mfe/build/mfe/Dockerfile
index 726d9ca..b996eea 100644
--- a/tutormfe/templates/mfe/build/mfe/Dockerfile
+++ b/tutormfe/templates/mfe/build/mfe/Dockerfile
@@ -3,7 +3,18 @@ FROM docker.io/node:12-bullseye-slim AS base
 RUN apt update \
   && apt install -y git \
     # required for cwebp-bin
-    gcc git libgl1 libxi6 make
+    gcc git libgl1 libxi6 make \
+    # additionally required for gifsicle, mozjpeg, and optipng
+    autoconf libtool pkg-config zlib1g-dev \
+    # additionally required for node-sass
+    python g++
+
+# Required for building optipng on M1
+ENV CPPFLAGS=-DPNG_ARM_NEON_OPT=0
+
+# Required for avoiding failed frontend-app-learning build due to missing binary on M1
+# (breaks pact tests)
+ENV PACT_SKIP_BINARY_INSTALL=true
 
 RUN mkdir -p /openedx/app /openedx/env
 WORKDIR /openedx/app
```

Unfortunately, using `PACT_SKIP_BINARY_INSTALL` is a hack that will break any Pact tests, so it's not really suitable to merge in. I don't see much alternative until there's an ARM binary for Pact though. Relevant issue: pact-foundation/pact-js-core#347

### PR DESCRIPTION
**Pull request by regisb**
Additional requirements are required, and we cannot download the pact binary on
arm64: https://github.com/pact-foundation/pact-js-core/issues/264

Close #31.

### CODE REVIEW

**Comment by bradenmacdonald** [comment]
@regisb 

I ran `tutor images build mfe` and it worked, and I can confirm the resulting image has the `arm64` architecture. 👍🏻

**Comment by jigsawlabs-student** [comment]
@regisb Unfortunately I continue to run into an error on this.  

`tutor images build mfe`

```
#49 73.51 > @pact-foundation/pact-node@10.13.8 postinstall /openedx/app/node_modules/@pact-foundation/pact-node
#49 73.51 > node postinstall.js
#49 73.51
#49 73.72 /openedx/app/node_modules/@pact-foundation/pact-node/standalone/install.js:41
#49 73.72     throw new Error(chalk_1.default.red("Error while installing binary: " + msg));
#49 73.72     ^
#49 73.72
#49 73.72 Error: Error while installing binary: Cannot find binary for platform 'linux' with architecture 'arm64'.
#49 73.72     at throwError (/openedx/app/node_modules/@pact-foundation/pact-node/standalone/install.js:41:11)
#49 73.72     at getBinaryEntry (/openedx/app/node_modules/@pact-foundation/pact-node/standalone/install.js:289:11)
```

**Comment by regisb** [comment]
Why is it working for @bradenmacdonald but not for @jigsawlabs-student? Are you quite sure that you checked out the "regisb/fix-arm-build" branch on your local Tutor repository? Did you also run `tutor config save && tutor images build mfe`?

**Comment by jigsawlabs-student** [comment]
Let me try again with checking out the fix arm branch..thanks for
clarifying that.

**Comment by regisb** [comment]
OK without further information I'll assume that this fixes image building on arm64.
